### PR TITLE
Fix up NSS links or delete

### PR DESCRIPTION
**Pull request by hamishwillee**
Continues: #2669

Fix up remaining NSS broken links. In some cases by finding new external links, in others by deleting the links. 

Also created VERY basic stub docs for the fundamental types. They are populated with the minimal info from the header, and link to the header. I have not tried to do this properly - these are just to fix links and give people a starting point (IMO if they are useful the NSS team should expand them, and if not, they should delete them and remove the inter-links)
- /en-US/docs/Mozilla/Projects/NSS/Reference/CK_RV
- /en-US/docs/Mozilla/Projects/NSS/Reference/CK_SESSION_HANDLE
- /en-US/docs/Mozilla/Projects/NSS/Reference/CK_SLOT_ID
- /en-US/docs/Mozilla/Projects/NSS/Reference/CK_MECHANISM
- /en-US/docs/Mozilla/Projects/NSS/Reference/CK_BYTE 
- /en-US/docs/Mozilla/Projects/NSS/Reference/CK_ULONG
- /en-US/docs/Mozilla/Projects/NSS/Reference/CK_OBJECT_HANDLE
- /en-US/docs/Mozilla/Projects/NSS/Reference/CK_ATTRIBUTE

### CODE REVIEW

File: files/en-us/mozilla/projects/nss/faq/index.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h3 id="What_platforms_and_development_environments_are_supported.3F">What platf
...
+<p>NSS supports <a href="/en-US/docs/Glossary/SSL">SSL v2 and v3</a>, <a href="/en-US/docs/Glossary/TLS">TLS</a>, <a href="https://tools.ietf.org/html/rfc8018">PKCS #5</a>, <a href="https://tools.ietf.org/html/rfc2315">PKCS #7</a>, <a href="https://www.oasis-open.org/committees/documents.php?wg_abbrev=pkcs11&show_descriptions=yes">PKCS #11</a>, <a href="https://tools.ietf.org/html/rfc7292">PKCS #12</a>, <a href="https://en.wikipedia.org/wiki/S/MIME">S/MIME</a>, and <a href="/en-US/docs/Mozilla/Security/x509_Certificates">X.509 v3</a> certificates. For complete details, see <a class="external" href="http://www-archive.mozilla.org/projects/security/pki/nss/nss-3.11/nss-3.11-algorithms.html">Encryption Technologies Available in NSS 3.11</a></p>
```
Comment (hamishwillee) [inline files/en-us/mozilla/projects/nss/faq/index.html:46]: Links to RFCS where they can be found. Changed S/MIME to wikipedia - not worth doing a "proper job" of a normal glossary entry.

**Comment by hamishwillee** [comment]
@chrisdavidmills This was a while ago now. Perhaps if they have already taken their snapshot we should just close?

**Comment by chrisdavidmills** [comment]
Crikey, I don't recall ever seeing this one. Sorry @hamishwillee . Yeah, we should just close this. The team have already been sent a snapshot of the docs.

**Comment by hamishwillee** [comment]
Yoiks. Let us never speak of this again :-0

**Pull request closed without merging**